src/Math/Helper.dfy

```diff
@@ -8,6 +8,27 @@ module Helper {
    Definitions
   ************/
 
+  function ArrayToSeq<T>(a: array<T>): (xs: seq<T>)
+    reads a
+    ensures a.Length == |xs|
+  {
+    a[..]
+  }
+
+  function NatToString(n: nat): string {
```

Comment:
We should probably do a feature request if this is not available.

Reply:
It's available in `Std.Strings`, but I'll resort to the custom one for now as we'd otherwise import too many axioms.